site/en/docs/webstore/best_practices/index.md

```diff
@@ -9,10 +9,33 @@ description: >
 This page has advice on how you should implement your app and list it in the store. As the store
 matures and we learn from developers' experiences, these recommendations will be updated.
 
-## Design a great app
+## Design a high-quality extension
+
+We encourage you to develop extensions that are of high quality. High-quality extensions conform to
+standards of performance, security, and user experience, as summarized by the following guidelines:
+
+* **Compliance:** Does the extension comply with our [program policies]? Extensions must not violate
+any of these policies.
+
+* **Manifest V3:** Is the extension built on Manifest V3? Manifest V3 reflects the aims of our
+  platform vision, and all High Quality extensions should use it. (See this
+  [MV3 overview][mv3-overview].)
+
+* **Security:** Is the extension safe for users? Make sure your extension does not pose security
+  threats, does not use deceptive installation tactics and meets Chrome Web Store's data privacy
+  requirements (see this [FAQ][user-data-faq]).
+
+* **Performance:** Does the extension function at an outstanding level? High-quality extensions
+  don't just perform their intended action, but perform at an extremely high level. 
+
+* **User experience:** Is the extension a joy to use? Make sure your extensions are easy to
+  install and manage, with a good-looking, intuitive and seamless user experience.
```

Comment:
```suggestion
* **User experience:** Is the extension a joy to use? The extension itself should provide a
  good-looking, intuitive, and seamless user experience while also respecting the end user's
  privacy.
```

I wanted to acknowledge that some extensions currently use content scripts to inject extension UI on all pages. While that may provide a user good user experience for some users, others may not be comfortable giving the extension that much data access.